QLExtension/Info.plist

```diff
@@ -116,6 +116,7 @@
 				<string>org.arduino.source</string>
 				<string>org.asm.source</string>
 				<string>org.bash.source</string>
+				<string>org.bazel.source</string>
```

Comment:
Is this needed? Why?

Reply:
The `QLExtension/Info.plist` is mandatory for the quicklook extension to manage the file.
The `Application/Info.plist` define the custom UTI properties (like the file extensions).